manuals/1.0/en/database.md

```diff
@@ -221,16 +596,39 @@ class Index
 }
 ```
 
+## Connect to multiple databases
+
+To connect to multiple databases, specify the identifier as the second argument.
+
+```php?start_inline
+$this->install(new DbalModule($logDsn, 'log_db');
+$this->install(new DbalModule($jobDsn, 'job_db');
+```
+
+```php?start_inline
+/**
+ * @Inject
+ * @Named("log_db")
+ */
+public function setLogDb(Connection $logDb)
+```
+
 [MasterSlaveConnection](http://www.doctrine-project.org/api/dbal/2.0/class-Doctrine.DBAL.Connections.MasterSlaveConnection.html) is provided for master/slave connections.
 
 # CakeDb
 
-**CakeDb** is the database access module for the CakePHP3 Database library. This module is provided by [@lorenzo](https://github.com/lorenzo) ( original author of CakeDb).
+**CakeDb** is an ORM using the active record and data mapper pattern idea. It is the same as the one provided in CakePHP3.
```

Comment:
I updated the CakeDb part based on what is on the Japanese version https://bearsunday.github.io/manuals/1.0/ja/database